packages/office-ui-fabric-react/src/components/Persona/Persona.scss

```diff
@@ -221,20 +222,20 @@ $ms-Persona-presenceSizeXl: 28px;
 }
 
 .ms-Persona-details {
+  box-sizing: border-box;
   padding: 0 12px;
   vertical-align: middle;
   @include text-align(left);
   @include padding-left($ms-Persona-sizeMd + $ms-Persona-imageDetailsLgSpace);
   display: table-cell;
+  width: 100%;
```

Comment:
Agreed, I'd love to completely refactor this with flexbox. I know @micahgodbolt expressed some interest as well. 😉  The Persona could be greatly simplified with flexbox, but it wouldn't be the easiest conversion.